docs/user_manual/C/gtg-plugins.page

```diff
@@ -39,6 +40,9 @@
 <p><em>Export and Print</em></p>
 <p>Exports the tasks in the current view into a variety of formats.</p>
 
+<p><em>Gamify</em></p>
+<p>Gamify adds a game aspect to your GTG workflow.</p>
```

Comment:
Maybe a short mention what "game aspect" means here? Like "(like task completion streaks)".

Reply:
Good point! Added a bit more of a description for this.